nav_msgs/Path   target_path
---
bool            is_traversed
---
nav_msgs/Path   remaining_path